.env_SAMPLE

```diff
@@ -125,6 +139,21 @@ export TOXENV=py27
 #export SAUCE_USERNAME=<sauce_username>
 #export SAUCE_ACCESS_KEY=<sauce_access_key>
 #export SAUCE_SELENIUM_URL=localhost:4445/wd/hub
+#export SAUCE_TUNNEL=<sauce_tunnel_id>
+
+#######################################################################
+# DO NOT EDIT VALUES. Project configuration - export variable defaults.
+#######################################################################
+
+export DJANGO_STAGING_HOSTNAME=${DJANGO_STAGING_HOSTNAME:-'content.localhost'}
```

Comment:
The [fish](https://fishshell.com/) shell [has issues](http://unix.stackexchange.com/q/81421) with this syntax:

```sh
~> export DJANGO_STAGING_HOSTNAME=${DJANGO_STAGING_HOSTNAME:-'content.localhost'}
${ is not a valid variable in fish.
```